skbio/io/tests/test_fasta.py

```diff
@@ -63,12 +64,16 @@ def setUp(self):
             'empty',
             'whitespace_only',
             'fasta_invalid_missing_header',
-            'fasta_invalid_blank_line',
```

Comment:
You renamed this file to `fasta_invalid_blank_line_in_record` but it doesn't look like it's used in the tests. It's also misnamed because the file should be valid (the blank line occurs between records).